### PR TITLE
RDoc-2924 Prefixed sharding

### PR DESCRIPTION
**Pull request by Danielle9897**
**Related issue:**
https://issues.hibernatingrhinos.com/issue/RDoc-2924/Documentation-for-Shard-By-Prefix

### CODE REVIEW

File: Documentation/6.1/Raven.Documentation.Pages/sharding/administration/sharding-by-prefix.markdown [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  * [Why use prefixed sharding](../../sharding/administration/sharding-by-prefix#why-use-prefixed-sharding)  
+  * [Overview](../../sharding/administration/sharding-by-prefix#overview)  
+  * [Bucket management](../../sharding/administration/sharding-by-prefix#bucket-management)  
+  * [Adding prefixes via Studio](../../sharding/administration/sharding-by-prefix#adding-prefixes-via-studio)
+  * [Adding prefixes via Client API](../../sharding/administration/sharding-by-prefix#adding-prefixes-via-client-api)
+      * [Add prefixes when creating a database](../../sharding/administration/sharding-by-prefix#add-prefixes-when-creating-a-database)
+      * [Add prefixes after database creation](../../sharding/administration/sharding-by-prefix#add-prefixes-after-database-creation)
+  * [Removing prefixes](../../sharding/administration/sharding-by-prefix#removing-prefixes)
+  * [Updating shard configurations for prefixes](../../sharding/administration/sharding-by-prefix#updating-shard-configurations-for-prefixes)
+  * [Prefixed sharding vs Anchoring documents](../../sharding/administration/sharding-by-prefix#prefixed-sharding-vs-anchoring-documents)
```
Comment (Danielle9897) [inline Documentation/6.1/Raven.Documentation.Pages/sharding/administration/sharding-by-prefix.markdown:28]: A section about: **Querying selected shards** will be added when the following issue is done:  https://issues.hibernatingrhinos.com/issue/RavenDB-22741/Prefixed-Sharding-Add-support-for-querying-by-prefix

File: Documentation/6.1/Raven.Documentation.Pages/sharding/querying.markdown [hunk 2]
```diff
@@ -0,0 +1,375 @@
+﻿# Sharding: Querying
+---
+
+{NOTE: }
+
+query specfic
+know in advance
```
Comment (aviv86) [inline Documentation/6.1/Raven.Documentation.Pages/sharding/querying.markdown:7]: remove this?

**Review by aviv86 — APPROVED** [review]

File: Documentation/6.1/Raven.Documentation.Pages/sharding/administration/images/assigned-bucket-ranges.png
[the remark this answers is not in the document]
Comment (Danielle9897) [reply Documentation/6.1/Raven.Documentation.Pages/sharding/administration/images/assigned-bucket-ranges.png:1]: fixed

File: Documentation/6.1/Raven.Documentation.Pages/sharding/querying.markdown (hunk 2 above)
Comment (Danielle9897) [reply Documentation/6.1/Raven.Documentation.Pages/sharding/querying.markdown]: removed

**Review by aviv86 — APPROVED** [review]

**Comment by ppekrol** [comment]
Can you rebase?

**Pull request merged**